src/clientiface.cpp

```diff
@@ -250,7 +267,9 @@ void RemoteClient::GetNextBlocks (
 			*/
 
 			f32 dist;
-			if (!isBlockInSight(p, camera_pos, camera_dir, camera_fov, d_blocks_in_sight, &dist)) {
+			if (!(isBlockInSight(p, camera_pos, camera_dir, camera_fov, d_blocks_in_sight, &dist) ||
+				(playerspeed.getLength() > 1.0f * BS &&
+				isBlockInSight(p, camera_pos, playerspeeddir, camera_fov / 10.0f, d_blocks_in_sight)))) {
```

Comment:
Maybe use a constant FOV angle here such as default FOV / 10? So that those who use wide or narrow FOV don't cause am unnecessarily wide or too-narrow velocity cone?

Reply:
I guess it would be (7.2f * M_PI / 180.0f) * 4.0f / 3.0f